native/balancers/mod_lbmethod_cluster.c

```diff
@@ -167,8 +167,8 @@ static int lbmethod_cluster_trans(request_rec *r)
     if (balancer) {
 
         /* It is safer to use r->uri */
-        if (strncmp(r->uri, "balancer://", 11)) {
-            r->filename = apr_pstrcat(r->pool, "proxy:balancer://", balancer, r->uri, NULL);
+        if (strncmp(r->uri, BALANCER_PREFIX, BALANCER_PREFIX_LENGTH)) {
+            r->filename = apr_pstrcat(r->pool, ("proxy:" BALANCER_PREFIX), balancer, r->uri, NULL);
```

Comment:
These parentheses are not needed, but I think it may help the clarity – there's a concatenation happening... WDYT?

Reply:
Yes they are here for clarity.